endpoints/auction.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/prebid/prebid-server/pbs"
 	"github.com/prebid/prebid-server/pbsmetrics"
 	pbc "github.com/prebid/prebid-server/prebid_cache_client"
+	"github.com/prebid/prebid-server/privacy"
+	gdprPolicy "github.com/prebid/prebid-server/privacy/gdpr"
```

Comment:
Aliases such as this for gdpr will go away over time as the `gdpr` package is rolled into `privacy/gdpr`.